sdk-base/src/main/java/com/mapbox/maps/plugin/animation/MapAnimationOptions.kt

```diff
@@ -43,6 +47,11 @@ class MapAnimationOptions private constructor(
      */
     private var duration: Long? = null
 
+    /**
+     * The amount of time, in milliseconds, to delay starting the animation after animation start.
+     */
+    private var startDelay: Long? = null
```

Comment:
should we think about setting this value to 0 and not to call animation to avoid memory leaks if user close/leave activity/fragment
or its responsibility of user itself?

Reply:
Users can get a `Cancelable`, they can cancel the animation by themselves.

Reply:
got it
should we add it as a reminder for users?